src/cpp/lpnamer/problem_modifier/CMakeLists.txt

```diff
@@ -21,7 +21,9 @@ add_library (lp_namer_problem_modifier STATIC
 	${CMAKE_CURRENT_SOURCE_DIR}/LauncherHelpers.h
 	${CMAKE_CURRENT_SOURCE_DIR}/MasterGeneration.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/MasterGeneration.h
-    )
+        ../model/Problem.cpp ../model/Problem.h
```

Comment:
theses src are added twice, see in /model.

May be consider adding them in /model/CMakeLists.txt  with relative path: 
`${CMAKE_CURRENT_SOURCE_DIR}Problem.h`
`${CMAKE_CURRENT_SOURCE_DIR}Problem.cpp`
then they will be visible in /problem_modifier lib

Reply:
Thank you, it's indeed better